<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="1312" started-at="2017-08-15T11:29:14Z" finished-at="2017-08-15T11:29:15Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="1312" started-at="2017-08-15T11:29:14Z" finished-at="2017-08-15T11:29:15Z">
      <class name="com.ian.sblog.dao.UserDaoTest">
        <test-method status="PASS" signature="unitilsBeforeClass()[pri:0, instance:com.ian.sblog.dao.UserDaoTest@59494225]" name="unitilsBeforeClass" is-config="true" duration-ms="153" started-at="2017-08-15T19:29:14Z" finished-at="2017-08-15T19:29:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- unitilsBeforeClass -->
        <test-method status="FAIL" signature="unitilsBeforeTestSetUp(java.lang.reflect.Method)[pri:0, instance:com.ian.sblog.dao.UserDaoTest@59494225]" name="unitilsBeforeTestSetUp" is-config="true" duration-ms="0" started-at="2017-08-15T19:29:14Z" finished-at="2017-08-15T19:29:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.ian.sblog.dao.UserDaoTest.findUserByUsernameAndPassword()]]>
              </value>
            </param>
          </params>
          <exception class="org.unitils.core.UnitilsException">
            <message>
              <![CDATA[Error inserting test data from DbUnit dataset for method public void com.ian.sblog.dao.UserDaoTest.findUserByUsernameAndPassword()]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.unitils.core.UnitilsException: Error inserting test data from DbUnit dataset for method public void com.ian.sblog.dao.UserDaoTest.findUserByUsernameAndPassword()
	at org.unitils.dbunit.DbUnitModule.insertDataSet(DbUnitModule.java:190)
	at org.unitils.dbunit.DbUnitModule$DbUnitListener.beforeTestSetUp(DbUnitModule.java:791)
	at org.unitils.core.Unitils$UnitilsTestListener.beforeTestSetUp(Unitils.java:273)
	at org.unitils.UnitilsTestNG.unitilsBeforeTestSetUp(UnitilsTestNG.java:62)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:653)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: org.unitils.core.UnitilsException: Error while executing DataSetLoadStrategy
	at org.unitils.dbunit.datasetloadstrategy.impl.BaseDataSetLoadStrategy.execute(BaseDataSetLoadStrategy.java:46)
	at org.unitils.dbunit.DbUnitModule.insertDataSet(DbUnitModule.java:342)
	at org.unitils.dbunit.DbUnitModule.insertDataSet(DbUnitModule.java:268)
	at org.unitils.dbunit.DbUnitModule.insertDataSet(DbUnitModule.java:187)
	... 29 more
Caused by: org.dbunit.dataset.NoSuchColumnException: bg_user.ID -  (Non-uppercase input column: ID) in ColumnNameToIndexes cache map. Note that the map's column names are NOT case sensitive.
	at org.dbunit.dataset.AbstractTableMetaData.getColumnIndex(AbstractTableMetaData.java:117)
	at org.dbunit.operation.AbstractOperation.getOperationMetaData(AbstractOperation.java:89)
	at org.dbunit.operation.AbstractBatchOperation.execute(AbstractBatchOperation.java:150)
	at org.dbunit.operation.CompositeOperation.execute(CompositeOperation.java:79)
	at org.unitils.dbunit.datasetloadstrategy.impl.CleanInsertLoadStrategy.doExecute(CleanInsertLoadStrategy.java:45)
	at org.unitils.dbunit.datasetloadstrategy.impl.BaseDataSetLoadStrategy.execute(BaseDataSetLoadStrategy.java:44)
	... 32 more
]]>
            </full-stacktrace>
          </exception> <!-- org.unitils.core.UnitilsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- unitilsBeforeTestSetUp -->
        <test-method status="SKIP" signature="findUserByUsernameAndPassword()[pri:0, instance:com.ian.sblog.dao.UserDaoTest@59494225]" name="findUserByUsernameAndPassword" duration-ms="15" started-at="2017-08-15T19:29:15Z" finished-at="2017-08-15T19:29:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- findUserByUsernameAndPassword -->
        <test-method status="PASS" signature="unitilsAfterTestTearDown(java.lang.reflect.Method)[pri:0, instance:com.ian.sblog.dao.UserDaoTest@59494225]" name="unitilsAfterTestTearDown" is-config="true" duration-ms="4" started-at="2017-08-15T19:29:15Z" finished-at="2017-08-15T19:29:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.ian.sblog.dao.UserDaoTest.findUserByUsernameAndPassword()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- unitilsAfterTestTearDown -->
      </class> <!-- com.ian.sblog.dao.UserDaoTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
